CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with new lines followed by square/curly brackets.
 * Fixed an issue where **graph** commands would not clean their temporary files properly, causing successive commands to fail.
 * Fixed an issue where an error log message changed the terminal color.
+* Fixed an issue where in some cases, splitting unified integrations / scripts with a period in their name would result in an incorrect filename being used for the code file.
```

Comment:
which cases? can it be more specific and still be a short log?

move it to `Unreleased`

Reply:
Updated it, let me know what if this is better.